.circleci/config.yml

```
@@ -279,6 +282,28 @@ jobs:
                     }
                   }'
 
+  wp-i18n-extract-strings:
+    working_directory: ~/wp-calypso
+    environment: *shared-environment
+    docker:
+      - image: php:7-cli
```

Let's try just using the wp cli image:

```suggestion
      - image: wordpress:cli
```

That should save us setting up the phar ourselves.